### PR TITLE
The last remaining cleanups for IDR demo3 (IDR-0.3.0)

### PR DESCRIPTION
**Pull request by manics**
A couple of remaining cleanups. All other commits have been moved to other PRs.

~~This is my branch with all changes planned for the IDR demo3 release. This is or will be split into separate PRs.~~

### CODE REVIEW

**Comment by snoopycrimecop** [comment]
Conflicting PR. Removed from build [MANAGEMENT_TOOLS-merge#1047](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1047/). See the [console output](https://ci.openmicroscopy.org/job/MANAGEMENT_TOOLS-merge/1047/consoleText) for more details.
Possible conflicts:
  - PR #150 manics 'Add external munin roles and idr munin playbook'
    - ansible/vendor/munin
    - ansible/idr-playbooks/management.yml
  - PR #151 manics 'Nginx idr caching config'
    - ansible/idr-playbooks/idr-omero.yml
    - ansible/idr-playbooks/group_vars/proxy-hosts.yml
  - PR #153 manics 'Use docker swarm with IDR JupyterHub'
    - ansible/idr-playbooks/idr-docker.yml
    - ansible/idr-playbooks/files/jupyterhub-docker-swarm.j2
  - PR #155 aleksandra-tarkowska 'upgrade django-redis'
    - ansible/idr-playbooks/files/IDR-OMERO-52-omero.j2
  - PR #157 manics 'omero user role'
    - ansible/roles/omero-user/README.md

**Comment by aleksandra-tarkowska** [comment]
could we add cookie name `omero.web.session_cookie_name sessionid_idr030` to avoid clashes?

**Comment by manics** [comment]
@aleksandra-tarkowska I've made the `idr_environment` var part of the cookie name: https://github.com/openmicroscopy/infrastructure/pull/159/commits/003046e91a030ed9cba13ae963bb0f75f03eb8eb

File: ansible/roles/omero-python-deps/tasks/pypi.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - libtiff-devel
+  - zlib-devel
+
+- name: omero python deps | pypi pillow
+  become: yes
+  pip:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - "Pillow==2.9"
```
Comment (aleksandra-tarkowska) [inline ansible/roles/omero-python-deps/tasks/pypi.yml]: could this go to `{{ omero_py_pip_dependencies }}` with an option `when: omero_py_pip_dependencies`

File: ansible/roles/omero-python-deps/tasks/pypi.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+# Install python modules from pypi
+
+- name: omero python deps | pip compilation requirements
+  become: yes
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+  - libjpeg-devel
```
Comment (aleksandra-tarkowska) [inline ansible/roles/omero-python-deps/tasks/pypi.yml]: I am guessing these could also be in vars

**Comment by joshmoore** [comment]
:+1: for cleaning up now inoperable playbooks.

**Pull request merged**